test/e2e/network/network_policy.go

```diff
@@ -821,49 +821,39 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			testCanConnect(f, nsB, "client-b", service, allowedPort)
 		})
 
-		ginkgo.It("should allow ingress access from updated pod [Feature:NetworkPolicy]", func() {
+		ginkgo.It("should deny ingress access to updated pod [Feature:NetworkPolicy]", func() {
 			const allowedPort = 80
-			ginkgo.By("Creating a network policy for the server which allows traffic from client-a-updated.")
+			ginkgo.By("Creating a network policy for the server which denies all traffic.")
 			policy := &networkingv1.NetworkPolicy{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "allow-pod-b-via-pod-selector",
+					Name: "deny-all",
```

Comment:
A better name might be `deny-ingress-via-isolated-label-selector`

Reply:
Done